server/athenian/api/controllers/default_controller.py

```diff
@@ -1,21 +1,41 @@
-from typing import List, Dict
+import datetime
+
 from aiohttp import web
+from sqlalchemy import select, sql
 
+from athenian.api import FriendlyJson
+from athenian.api.models import CalculatedMetric, CalculatedMetricValues
 from athenian.api.models.calculated_metrics import CalculatedMetrics
-from athenian.api.models.invalid_request_error import InvalidRequestError
+from athenian.api.models.db import github
+# from athenian.api.models.invalid_request_error import InvalidRequestError
 from athenian.api.models.metrics_request import MetricsRequest
-from athenian.api.models.no_source_data_error import NoSourceDataError
-from athenian.api import util
+# from athenian.api.models.no_source_data_error import NoSourceDataError
 
 
 async def calc_metrics(request: web.Request, body) -> web.Response:
     """Calculate metrics.
 
-    
-
     :param body: Desired metric definitions.
     :type body: dict | bytes
-
     """
     body = MetricsRequest.from_dict(body)
-    return web.Response(status=200)
+    github_repos_query = []
+    for for_set in body._for:
+        for repo in for_set.repositories:
+            if repo.startswith("github.com/"):
+                github_repos_query.append(github.Repository.name == repo[len("github.com/"):])
+    github_repos_query = select([github.Repository]).where(sql.or_(*github_repos_query))
+    repos = await request.db.fetch_all(query=github_repos_query)
+    print(repos)
+    met = CalculatedMetrics()
+    met.date_from = body.date_from
+    met.date_to = body.date_to
+    met.granularity = body.granularity
+    met.metrics = body.metrics
+    met.calculated = []
+    now = datetime.date.today()
+    for for_set in body._for:
+        met.calculated.append(CalculatedMetric(
+            _for=for_set,
+            values=[CalculatedMetricValues(date=now, values=[0.5] * len(met.metrics))] * 10))
```

Comment:
why 10?, why 0.5?

Reply:
but what does it mean?

Reply:
Yep, these are just random numbers for now.

The meaning is to return something as the metric values.